shared/src/database/services/donation-certificate/DonationCertificateWriter.ts

```diff
@@ -0,0 +1,162 @@
+import assert from 'assert';
+import { createWriteStream } from 'fs';
+import _ from 'lodash';
+import * as path from 'path';
+import PDFDocument from 'pdfkit';
+import { LANGUAGE_CODES } from '../../../types/language';
+import { Translator } from '../../../utils/i18n';
+import { ContributionDonationEntry } from '../contribution/contribution.types';
+import { ContributorDonationCertificate } from '../contributor/contributor.types';
+
+const ASSET_DIR = path.join(process.cwd(), 'public', 'assets');
+
+export class DonationCertificateWriter {
```

Comment:
pls rename the file from DonationCertificateWriter to donation-certificate-writer